.github/workflows/ci.yml

```diff
@@ -22,7 +22,25 @@ jobs:
       # and hyphens.
       run: echo "TAG=branch-${GITHUB_REF_NAME//[^A-Za-z0-9._-]/-}" | tee -a $GITHUB_ENV
 
-    - run: ./devel/build $TAG
+    - uses: docker/setup-qemu-action@v2
+
+      # Adapted from https://github.com/actions/runner-images/issues/6180#issuecomment-1236674387
+    - name: Install newer version of skopeo
+      run: |
+        sudo apt purge golang-github-containers-common skopeo
+        sudo apt autoremove --purge
+        REPO_URL="https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/unstable"
+        source /etc/os-release
+        sudo sh -c "echo 'deb ${REPO_URL}/x${NAME}_${VERSION_ID}/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:unstable.list"
+        sudo wget -qnv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/x${NAME}_${VERSION_ID}/Release.key -O Release.key
+        sudo apt-key add Release.key
+        sudo apt-get update
+        sudo apt-get install skopeo
+        skopeo --version
```

Comment:
This is a lot. Can we run a [containerized Skopeo](https://github.com/containers/skopeo/blob/main/install.md#container-images) instead? e.g.

```
docker run --rm -it quay.io/skopeo/stable --version
```

Reply:
Good idea, done: 1769212f6fa8b31e86da07cbe6391c87cfc9c44a

Reply:
Might need adjustment to pass in credentials for docker.io to the skopeo container?

Reply:
Oh that makes sense... probably need to pass `secrets.DOCKER_LOGIN` and `secrets.DOCKER_PASSWORD` when calling the script:

https://github.com/nextstrain/docker-base/blob/70443f60fa6d33f9e16780db6ca5dbbd491f3a1b/.github/workflows/ci.yml#L53

I think 4 arguments calls for using named args with `getopt`, or expect them to be available as env vars.

Reply:
Yeah, they should be env vars, and I expect they should be passable to skopeo's container as env vars too.

Reply:
Ah, looks like it doesn't support passing via env vars ([ref](https://github.com/containers/skopeo/blob/main/docs/skopeo-copy.1.md)), but you can have it read the Docker auth file by mounting it into the container with:

```
-v $HOME/.docker/config.json:/tmp/auth.json:ro
```

or a bit more scoped:

```
-v $HOME/.docker/config.json:/docker-auth.json:ro … --dest-authfile /docker-auth.json
```

Reply:
Done in f83877f6320b22bb4d6e5c449fdb4d89c73990a1:

https://github.com/nextstrain/docker-base/blob/f83877f6320b22bb4d6e5c449fdb4d89c73990a1/devel/localhost-to-dockerhub#L30-L43